tests/cross/src/test/scala/tests/pc/CompletionArgSuite.scala

```diff
@@ -114,7 +143,14 @@ class CompletionArgSuite extends BaseCompletionSuite {
        |age = : Int
        |followers = : Int
        |""".stripMargin,
-    topLines = Option(3)
+    topLines = Option(3),
+    compat = Map(
+      "3.0" ->
+        """|address = : String
```

Comment:
It looks the same as the default, so maybe compat is not needed? I think I noticed it in a couple of other places

Reply:
Ah, that's correct, fixed in  https://github.com/scalameta/metals/pull/2369/commits/8d2a018299384c70dff7e64dc64edcf16d2b3a52